04 Time to Practice - Component Basics/01-starting-setup/src/components/Concepts/Concepts.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import ConceptItems from "./ConceptItems";
+import './Concepts.css';
+
+const Concepts = (props) =>{
+    return(
+        <ul id="concepts">
+            <ConceptItems 
```

Comment:
Can you iterate here and generate the ConceptItems instead of using it individually in three child ConceptItems